<launch>
  <arg name="always_run" default="0" />
  <node 
    pkg="mrobosub_perception"
    type="pathmarker_hsv.py"
    name="pathmarker_hsv"
    output="screen"
    args="$(arg always_run)"
  >
    <rosparam command="load" file="$(find mrobosub_perception)/params/pathmarker_hsv.yaml" />
  </node> 
</launch>
